ui-participant/src/hub/HubPage.tsx

```diff
@@ -82,6 +83,9 @@ export default function HubPage() {
           style={{ background: '#fff', maxWidth: 768 }}
         >
           {relations.length > 0 && <HubPageParticipantSelector/>}
+          {enrollees.length === 0 && <div className={'d-flex justify-content-center'}>
+            <LoadingSpinner/>
+          </div>}
```

Comment:
Might not be the best way to handle this - I fear if someone gets into a hanging state without an enrollee, they will be confused about the fact that it seems to be loading infinitely.

Reply:
I am honestly not sure what is happening here. It's kind of hard to debug. With the "await" added on the study enroll router, it should actually be fully post-login/enrollment, so I don't think there's any actual network things happening when the loading indicator happens right now. I think it is just that react needs a couple frames to refresh internal state to populate the enrollee property on the active user provider. So, maybe it doesn't make sense to add a loading indicator? It will flash, but it won't flash "All activities complete" anymore.